playbooks/molecule/centos7_monitoring/molecule.yml

```diff
@@ -25,6 +25,7 @@ platforms:
       - all
       - monitoring_client
       - monitoring_service
+      - centos7
```

Comment:
Is there a reason not to use the `check_default_version` task in the `install_python` role instead of adding the `centos7` and `rocky9` groups (which are in both `resources/monitoring` and `resources/xnat`) to set the `install_python` var?

Reply:
I think @paddyroddy was having issues with those pre-tasks when trying to use a different image for testing? And I thought it seemed easier to follow to have variables defined in the group vars rather than tasks that set them, but happy to change it

Reply:
> I think @paddyroddy was having issues with those pre-tasks

Ah, I didn't realise. It's just that the group vars are duplicated for both sets of playbook tests.

Reply:
ah yeah I thought about that. We could have a single inventory for both, and combine the scenarios so we have `centos7` and `rocky9` scenarios rather than playbook-specific ones. Then in the converge etc. playbook etc. tag the tasks like we were doing for the roles. But that felt like a lot of changes for an already large pr!